api/rql/ast/predicateExpression.go

```diff
@@ -0,0 +1,46 @@
+package ast
+
+import (
+	"time"
+
+	"github.com/puppetlabs/wash/api/rql"
+	"github.com/puppetlabs/wash/api/rql/internal/predicate"
+	"github.com/puppetlabs/wash/api/rql/internal/predicate/expression"
+	"github.com/puppetlabs/wash/plugin"
+	"github.com/shopspring/decimal"
+)
+
+// PE_Primary returns a node representing PE Primary
```

Comment:
PE wasn't obvious to me, maybe write out `predicate expression primary` in the comment.

Reply:
Just needs to be spelled out the first time it's used in a file.

Reply:
Agreed. I'll update.

Reply:
Updated.